src/ui/screens/devices/ViewDeviceLimit.qml

```diff
@@ -19,15 +19,15 @@ MZViewBase {
 
             property bool skipEnsureVisible: true
 
-            labelText: MZI18n.GetHelpLinkTitle
+            labelText: MZI18n.GetHelpLinkText
             onClicked: {
                 Glean.sample.getHelpClickedInitialize.record();
                 MZNavigator.requestScreen(VPN.ScreenGetHelp);
             }
         }
     }
 
-    //% "My devices"
+    //% "Devices"
     _menuTitle: qsTrId("vpn.devices.myDevices")
```

Comment:
Realized only today that this and another string were changed without a new ID, so localization will reflect the English text only from the next version of VPN 😞